### DISCUSSION TITLE
CRUD example - dropdown when selecting a state (U.S states)

### DISCUSSION THREAD

**Discussion by unfor19** [KevinVandy/material-react-table#430] [Q&A]
Hi,
When editing a row, everything is perfect; the `State` dropdown behaves as expected, though when I click on **CREATE NEW ACCOUNT**, there's no dropdown for `State`, and **I can insert free text**.

Can someone please share how to enable the dropdown to **create row** part? I've tried a few things, but nothing works.

Link to CRUD example - [https://www.material-react-table.com/docs/examples/editing-crud](https://www.material-react-table.com/docs/examples/editing-crud)

Attached is a video of what I mean:

https://user-images.githubusercontent.com/15122452/225562726-ebf30258-3fdc-4014-b6bc-03e5a10905c6.mov

P.S- I need something that supports both `multiselect=true` and "single select" `multiselect=false`

Here's an example from material-table that looks legit - [https://github.com/mbrn/material-table/issues/2219#issuecomment-662581333](https://github.com/mbrn/material-table/issues/2219#issuecomment-662581333)

**Comment by KevinVandy** [comment]
The example needs to be updated. I think the Create Modal code in that example is just kinda wrong
